joschahenningsen/TUM-Live
# Feature/bookmarks in seekbar

### Motivation and Context
It would be nice to see the bookmarks in the seekbar, to provide the user a glimpse of his bookmarks.

### Description
Bookmark marker are added to the seekbar.

### Steps for Testing
Prerequisites:
- 1 Student
- 1 Livestream

1. Log in
2. Navigate to a Livestream
3. Create a bookmark and check if the bookmark appears in the seekbar
4. Reload the page and check if the bookmark is still visible in the seekbar
5. Remove a bookmark and check if the bookmark is removed from the seekbar

### Screenshots
![Bildschirmfoto 2023-04-04 um 01 05 05](https://user-images.githubusercontent.com/17506411/229645600-6ffc8d08-f00b-41e2-9138-08fcde99a119.png)

![Bildschirmfoto 2023-04-04 um 23 38 21](https://user-images.githubusercontent.com/17506411/229930993-27c448c0-1e5f-4ec7-8ff9-098766d5a1f5.png)


https://user-images.githubusercontent.com/17506411/230999459-b245c1c4-0862-4435-ad53-979d851f3cbb.mov
